config.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+
+const config = {
+  server: {
+    hostname: process.env.SERVER_HOSTNAME,
+    port: process.env.SERVER_PORT,
+  },
+  jwt: {
+    secret: process.env.JWTSECRET,
+  },
+  database: {
+    url: process.env.DATABASE_URL,
```

Comment:
the db url is defined as MONGODB. i'll give you the .env file when I get the chance.